modules/olm-crds.adoc

```diff
@@ -31,4 +31,8 @@ The following custom resource definitions (CRDs) are defined and managed by Oper
 |`OperatorGroup`
 |`og`
 |Configures all Operators deployed in the same namespace as the `OperatorGroup` object to watch for their custom resource (CR) in a list of namespaces or cluster-wide.
+
+|`OperatorConditions`
+|-
+|Creates a communication channel between OLM and an Operator it manages. Operators can write to the `Status.Conditions` array to modify OLM management of the Operator.
```

Comment:
This feature could be used to highlight information that does not modify OLM management of the operator. 
```suggestion
|Creates a communication channel between OLM and an Operator it manages. Operators can write to the `Status.Conditions` array to communicate complex state to OLM.
```